base/deprecated.jl

```diff
@@ -1732,6 +1732,9 @@ end
 @deprecate mapfoldl(f, op, v0, itr) mapfoldl(f, op, itr; init=v0)
 @deprecate mapfoldr(f, op, v0, itr) mapfoldr(f, op, itr; init=v0)
 
+# PR #27856
+@eval Sys @deprecate_binding CPU_CORES CPU_LOGICAL_CORES false
```

Comment:
@fredrikekre did as you suggested but the CI build is saying that `@deprecate_binding` is not defined

Reply:
`Base.@deprecate_binding`